README.md

```diff
@@ -15,7 +19,7 @@ mkvirtualenv -p /usr/local/bin/python3 notifications-api
 
 Creating the environment.sh file. Replace [unique-to-environment] with your something unique to the environment. Your AWS credentials should be set up for notify-tools (the development/CI AWS account).
 
-Create a local environment.sh file containing the following:
+Create a local environment.sh file containing the following, updating the AWS key entries with your own values:
```

Comment:
dont think you need this addition - since you have to update other entries (eg mmg/firetest API keys) it should be apparent that they should be updated. happy to keep it if you want to reword it to not explicitly mention AWS tho